corehq/apps/userreports/reports/filters/values.py

```diff
@@ -128,6 +128,37 @@ def to_sql_values(self):
         }
 
 
+class PreFilterValue(FilterValue):
+
+    def is_null(self):
+        return self.value is None
+
+    def is_list(self):
+        return isinstance(self.value, list)
```

Comment:
are `is_null` and `is_list` meant to be used external to the class? if not it would be clearer to prefix them with `_`.